ui/admin/app/components/worker-diagram/dual-filter/index.hbs

```diff
@@ -25,7 +25,11 @@
         {{t 'resources.target.workers.diagram.client'}}
       </D.Item>
       <D.Item @icon='settings'>
-        {{t 'resources.target.workers.diagram.front-line-worker'}}
+        {{#if (feature-flag 'target-worker-filters-v2-hcp')}}
+          {{t 'resources.target.workers.diagram.ingress-worker'}}
+        {{else}}
+          {{t 'resources.target.workers.diagram.front-line-worker'}}
+        {{/if}}
```

Comment:
I decided to go this route vs WorkerDiagram::DualFilter::HCP as it reduces duplicate code. Happy to create an HCP specific component if the team feels it is a better route.